packages/preact-flicking/src/preact-flicking/Flicking.js

```diff
@@ -0,0 +1,3 @@
+import Flicking from "@egjs/react-flicking";
+
+export default Flicking;
```

Comment:
I think there could be a reason to make this file in `js` format, can I ask why?

Reply:
@WoodNeck 

One of the ways to convert from react to preact is to remove the react type.
So, I created Flicking.js and Flicking.d.ts.  React was removed from Flicking.d.ts and changed to preact.